From db68d2f13471d435fd68493e9ccd18a581167f02 Mon Sep 17 00:00:00 2001
From: Sven Schwermer <sven.schwermer@disruptive-technologies.com>
Date: Mon, 27 Dec 2021 12:51:03 +0100
Subject: [PATCH 1/3] mys_6ulx: Unset initrd_high in environment

This makes u-boot copy the RAM disk to the end of the memory region. Not
quite sure why Linux detects an overlap when it's booted with the RAM
disk in place (where the FIT image is loaded).
---
 include/configs/mys_6ulx.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/configs/mys_6ulx.h b/include/configs/mys_6ulx.h
index 5ef16fb278..bdb8908371 100644
--- a/include/configs/mys_6ulx.h
+++ b/include/configs/mys_6ulx.h
@@ -56,7 +56,6 @@
 	"mtdparts=" CONFIG_MTDPARTS_DEFAULT "\0" \
 	"fdt_addr_r=0x82000000\0" \
 	"fdt_high=0xffffffff\0" \
-	"initrd_high=0xffffffff\0" \
 	"kernel_addr_r=0x81000000\0" \
 	"pxefile_addr_r=0x87100000\0" \
 	"ramdisk_addr_r=0x82100000\0" \
-- 
2.34.1

